### PR TITLE
Fix for ballerina version changes (Ballerina 0.970.0-beta3)

### PR DESCRIPTION
**Pull request by erandiganepola**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,21 @@
...
+[Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_what_is_rest_api.htm). 
+It handles [OAuth2.0](http://tools.ietf.org/html/rfc6749), provides auto completion and type safety.
 
 ## Compatibility
 
 | Ballerina Version         | Endpoint Version          | API Version |
 | ------------------------- | ------------------------- | ------------|
-|   0.970.0-beta1-SNAPSHOT  |          0.9.10           |   v37.0     |
-
+|   0.970.0-beta3           |          0.9.11           |   v37.0     |
```
Comment (muthulee) [inline README.md]: Please remove Ballerina version

File: README.md [hunk 2]
```diff
@@ -48,6 +30,18 @@ Do not share your access token, client secret with anyone.
...
+
+4. Import the Salesforce package(sfdc37) to your Ballerina program as follows.
+
+```ballerina
+
+import wso2/sfdc37;
+
+```
+
 ### Working with Salesforce REST endpoint.
```
Comment (muthulee) [inline README.md]: Use connector. and remvoe fullstop.

File: sfdc37/Package.md [hunk 3]
```diff
@@ -1,56 +1,69 @@
-# Salesforce Endpoint
+# Salesforce Connector
```
Comment (Biruntha) [inline sfdc37/Package.md]: Keep it as "Endpoint"

File: sfdc37/Package.md [hunk 4]
```diff
@@ -1,56 +1,69 @@
...
-of the [REST API](https://developer.salesforce.com/page/REST_API).
+## Compatibility
 
-Ballerina is a strong and flexible language. Also it is JSON friendly. 
-It provides an integration tool which can be used to integrate the Salesforce API with other endpoints. 
-It is easy to write programs for the Salesforce API by having an endpoint for Salesforce. 
-Therefor it allows you to access the Salesforce REST API through Ballerina easily. 
+| Ballerina Version         | Endpoint Version          | API Version |
+| ------------------------- | ------------------------- | ------------|
+|   0.970.0-beta3           |          0.9.11           |   v37.0     |
```
Comment (Biruntha) [inline sfdc37/Package.md]: Here we don't need to keep "Endpoint Version"

File: sfdc37/tests/Readme.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 | Ballerina Version         | Endpoint Version          | API Version |
 | ------------------------- | ------------------------- | ------------|
-|  0.970.0-beta1-SNAPSHOT   |           0.9.10          |   v37.0     |
+|  0.970.0-beta3            |           0.9.11          |   v37.0     |
```
Comment (Biruntha) [inline sfdc37/tests/Readme.md]: Remove "Endpoint Version" version

File: README.md (hunk 1 above)
Comment (Biruntha) [inline README.md]: Remove "Endpoint Version" from here

**Pull request merged**